framework/contracts/native/ibc-client/src/anybuf.rs

```diff
@@ -0,0 +1,145 @@
+use anybuf::Anybuf;
```

Comment:
Are we sure this removes a lot of wasm size ? Doesn't seem particularly healthy to provide our own ibc-proto types

Reply:
Yep, a lot, with opt-level =3 wasm don't fit, but with this change we have extra ~20 kb (It's still not enough if we ever decide to add any method)